rules/aip0136/declarative_standard_methods_only.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0136
+
+import (
+	"strings"
+
+	"bitbucket.org/creachadair/stringset"
+	"github.com/googleapis/api-linter/lint"
+	"github.com/googleapis/api-linter/rules/internal/utils"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var standardMethodsOnly = &lint.MethodRule{
+	Name:   lint.NewRuleName(136, "declarative-standard-methods-only"),
+	OnlyIf: utils.IsDeclarativeFriendlyMethod,
+	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
+		// Standard methods are fine.
+		standard := stringset.New("Get", "List", "Create", "Update", "Delete", "Undelete")
```

Comment:
Should the batch methods be included here?

Reply:
Oh, probably.

Reply:
Done.